server/api/person/person.subscribe.js

```diff
@@ -28,10 +33,28 @@ module.exports = (server) => {
     PubSub.publish(TOPIC_PERSON__EMAIL_SENT, info)
   })
 
+  PubSub.subscribe(TOPIC_PERSON__CREATE, async (msg, person) => {
+    const coreOrganisation = await Organisation.findOne({ name: CORE_ORGANISATION_NAME }).lean().exec()
+
+    if (!coreOrganisation) {
+      return
+    }
+
+    await addMember({
+      person: person.id.toString(),
+      organisation: coreOrganisation._id.toString(),
+      validation: SYSTEM_MEMBER_VALIDATION_MESSAGE
+    })
+  })
+
   PubSub.subscribe(TOPIC_MEMBER__UPDATE, async (msg, member) => {
     // a new member has been created or a member status has changed
     // send email to let people know
 
+    if (isSystemOperation(member.validation)) {
```

Comment:
function not really needed here
if (member.validation === SYSTEM_MEMBER_VALIDATION_MESSAGE)) return 

would suffice

Reply:
Done